tests/periph_systick/app.config.test

```diff
@@ -0,0 +1,5 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+CONFIG_MODULE_TEST_UTILS_INTERACTIVE_SYNC=y
```

Comment:
Not needed.

Reply:
ah that was it - should probably add an automated test though